pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -158,6 +158,12 @@ Before running the following command, ensure that your `dkp` configuration **ref
     ...
     ```
 
+1.  For air-gapped deployments, an additional step is required to upgrade the Grafana Loki MinIO Tenant
```

Comment:
```suggestion
1.  For air-gapped deployments, an additional step is required to upgrade the Grafana Loki MinIO Tenant
```
Should this be a step 2 as it is now?  Or should it be another bullet or maybe a NOTE under the step 1. above regarding air-gapped?  Not sure, just a suggestion.

Reply:
I can definitely move it up, i was on the fence since its a bit crowded up there in step 1. let me see how it looks!

Reply:
actually, thinking about this more... this would be an "ordered" step so having it under step 1 in an unordered list feels odd, especially with the two air-gapped bullet points

Reply:
Cool.. just needed verification for my own sanity.  I understand your point.  : )

Reply:
I needed to think it through for myself as well, so thanks for bring it up!